diff -Naur mesa-17.3.6/src/util/Makefile.am mesa-17.3.6.tpg/src/util/Makefile.am
--- mesa-17.3.6/src/util/Makefile.am	2018-02-27 00:21:09.000000000 +0000
+++ mesa-17.3.6.tpg/src/util/Makefile.am	2018-03-02 18:03:23.089572671 +0000
@@ -54,7 +54,7 @@
 	$(PTHREAD_LIBS) \
 	$(CLOCK_LIB) \
 	$(ZLIB_LIBS) \
-	$(LIBATOMIC_LIBS)
+	$(LIBATOMIC_LIBS) -lsupc++
 
 libxmlconfig_la_SOURCES = $(XMLCONFIG_FILES)
 libxmlconfig_la_CFLAGS = \
