--- mesa-26.2.0-rc2/src/intel/shaders/meson.build	2026-07-22 15:37:39.000000000 +0200
+++ mesa-patched/src/intel/shaders/meson.build	2026-07-23 17:46:54.626749073 +0200
@@ -55,6 +55,7 @@ foreach gen : intel_shaders_gens
       '-I' + join_paths(dir_source_root, 'src/intel'),
       '-I' + join_paths(meson.project_build_root(), 'src/intel'),
       '-I' + join_paths(dir_source_root, 'src/intel/genxml'),
+      '-I/usr/lib64/clang/23/include',
       cl_args,
     ],
     depends : [gen_cl_xml_pack, intel_dev_wa_src],
--- mesa-26.2.0-rc2/src/compiler/clc/meson.build	2026-07-22 15:37:39.000000000 +0200
+++ mesa-patched/src/compiler/clc/meson.build	2026-07-23 17:46:54.626815863 +0200
@@ -103,7 +103,7 @@ _libmesaclc = static_library(
   'libmesaclc',
   files_libmesaclc,
   sources: _libmesaclc_sources,
-  include_directories : [inc_include, inc_src, inc_spirv],
+  include_directories : [inc_include, inc_src, inc_spirv, '/usr/lib64/clang/23/include'],
   c_args : _libmesaclc_c_args,
   cpp_args : [_libmesaclc_cpp_args, _libmesaclc_c_args],
   dependencies: [idep_nir, dep_clang, dep_llvm, dep_llvmspirvlib,
--- mesa-26.2.0-rc2/src/intel/blorp/meson.build	2026-07-22 15:37:39.000000000 +0200
+++ mesa-patched/src/intel/blorp/meson.build	2026-07-23 17:46:54.626865484 +0200
@@ -15,6 +15,7 @@ blorp_shaders_spv = custom_target(
     '-I' + join_paths(meson.project_source_root(), 'src/compiler/libcl'),
     '-I' + join_paths(meson.project_source_root(), 'src'),
     '-I' + join_paths(meson.current_source_dir(), '.'),
+    '-I/usr/lib64/clang/23/include',
     cl_args,
     ],
   depends : [gen_cl_xml_pack],
@@ -55,7 +56,7 @@ files_blorp_elk = files(
 libblorp = static_library(
   'blorp',
   [files_libblorp, files_blorp_brw],
-  include_directories : [inc_include, inc_src, inc_intel, inc_intel_compiler],
+  include_directories : [inc_include, inc_src, inc_intel, inc_intel_compiler, '/usr/lib64/clang/23/include'],
   c_args : [no_override_init_args, soong_compat_c_args],
   gnu_symbol_visibility : 'hidden',
   dependencies : [
@@ -74,7 +75,7 @@ if with_intel_elk
   libblorp_elk = static_library(
     'blorp_elk',
     [files_libblorp, files_blorp_elk],
-    include_directories : [inc_include, inc_src, inc_intel],
+    include_directories : [inc_include, inc_src, inc_intel, '/usr/lib64/clang/23/include'],
     c_args : [no_override_init_args, soong_compat_c_args],
     gnu_symbol_visibility : 'hidden',
     dependencies : [
