diff -up mesa-24.1.4/meson.build.10~ mesa-24.1.4/meson.build
--- mesa-24.1.4/meson.build.10~	2024-08-14 15:14:40.055432887 +0200
+++ mesa-24.1.4/meson.build	2024-08-14 15:15:02.651681596 +0200
@@ -1850,7 +1850,7 @@ pre_args += '-DDRAW_LLVM_AVAILABLE=' + (
 if with_clover_spirv or with_clc
   chosen_llvm_version_array = dep_llvm.version().split('.')
   chosen_llvm_version_major = chosen_llvm_version_array[0].to_int()
-  chosen_llvm_version_minor = chosen_llvm_version_array[1].to_int()
+  chosen_llvm_version_minor = 0 # chosen_llvm_version_array[1].to_int()
 
   # Require an SPIRV-LLVM-Translator version compatible with the chosen LLVM
   # one.
